packages/react/src/components/CodeSnippet/CodeSnippet.stories.js

```diff
@@ -220,3 +220,32 @@ export const Skeleton = () => (
     <CodeSnippetSkeleton type="multi" />
   </div>
 );
+
+export const Playground = (args) => (
+  <CodeSnippet feedback="Copied to clipboard" {...args}>
+    {'node -v'}
+  </CodeSnippet>
+);
+
+Playground.argTypes = {
+  ariaLabel: {
+    table: {
+      disable: true,
+    },
+  },
+  children: {
+    table: {
+      disable: true,
+    },
+  },
+  onClick: {
+    table: {
+      disable: true,
```

Comment:
Same thing here, we can define the callback handlers in the component level argTypes and to get them logged out to the `Actions` panel